configs/sample_yamls/serviceops/sample_service_config.yml

```diff
@@ -0,0 +1,172 @@
+#
+########################## types ########################
+#
+types:
+# QUOTA
+- name: ratelimit_by_service_by_consumer
+  quota_params:
```

Comment:
Fixed my indenting above, wasn't suggesting we get rid of the outer 'types' / 'constructors' but I was suggesting that sequences of types are kept together.

Not a strong opinion btw, just want to make sure we have some natural readability